test/common/testlib.py

```diff
@@ -93,6 +93,21 @@
 opts.jobs = 1
 opts.fetch = True
 
+default_layouts = [
+    {
+        "name": "desktop",
+        "is_mobile": False,
+        "shell_size": [1920, 1200],
+        "content_size": [1680, 1130]
+    },
+    {
+        "name": "mobile",
+        "is_mobile": True,
+        "shell_size": [414, 1920],
+        "content_size": [414, 1856]
```

Comment:
Here it's even more obvious -- for such narrow widths we expect both widths to be equal, there is no freedom here.